examples/hal_cpu_remote_server/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) Codeplay Software Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License") with LLVM
+# Exceptions; you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+
+project(hal_cpu_remote_server VERSION 1.0.0 LANGUAGES C CXX ASM)
+
+set(HAL_CPU_REMOTE_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../clik/external/hal_cpu)
```

Comment:
Can we move `examples/clik/external/hal_cpu` to `examples/external/hal_cpu` since it is no longer limited to `clik`?

Reply:
That's a bigger change since the original idea of clik was that it was fairly standalone. I think it's a good idea, but I'd prefer to to do it in a separate PR.